dev/com.ibm.ws.messaging.open_clientcontainer_fat/bnd.bnd

```diff
@@ -27,7 +27,8 @@ tested.features: \
         messagingClient-3.0,\
         messagingServer-3.0,\
         messaging-3.0,\
-        jakartaeeClient-9.0
+        jakartaeeClient-9.0,\
+        xmlWS-3.0
```

Comment:
Since xmlWS-3.0 isn't a thing yet that can be really enabled, does this cause test failures?

Reply:
it caused test failures without it... my guess is that the FAT test uses stuff from the APIs, but not the impl itself.